### DISCUSSION TITLE
How can I add a dropdown in sidebar/navbar?

### DISCUSSION THREAD

**Discussion by akshit-arora** [filamentphp/filament#6118] [Help] [answered]
I want to have a drop-down which will load the other modules related to what was selected in the dropdown.

### Use Case:

My project has modules which are based on the selected option in the dropdown.

What I was able to find in the documentation was [this] (https://filamentphp.com/docs/2.x/admin/navigation#registering-custom-navigation-items)

### Example

For an application showing my products, orders, employees for my warehouse, I want ability to add multiple warehouses and select the warehouse from the sidebar/navbar to look into the relevant modules

**Comment by Wulfheart** [comment]
I have a similar use case and am really looking forward to any suggestions.

**Comment by danharrin** [comment, marked as answer]
`sidebar.start` Render hook is the easiest route - https://filamentphp.com/docs/2.x/admin/appearance#render-hooks

Then remove all navigation items and replace with your own code

**Reply by akshit-arora** [reply]
Thank you! This worked
